### PR TITLE
add global fastmath removal to news.

### PR DESCRIPTION
**Pull request by oscardssmith**
followup to https://github.com/JuliaLang/julia/pull/41638

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Command-line option changes
 
 * In Linux and Windows, `--threads=auto` now tries to infer usable number of CPUs from the
   process affinity which is set typically in HPC and cloud environments ([#42340]).
+* `--math-mode=fast` is now a no-op ([#41638]). local fastmath annotations (eg `@fastmath`)
+  give the same performance without breaking your code as much.
```
Comment (ViralBShah) [inline NEWS.md:28]: The statement "without breaking your code as much" sounds a bit cryptic and confusing. My suggestion is to either explain better or not refer to breakage and let people read the linked issue/PR. This reads conversational and casual and not like release notes.

Comment (simeonschaub) [reply NEWS.md:28]: Perhaps replace the second sentence with "Users are encouraged to use the `@fastmath` macro instead, which has more well-defined semantics."?

File: NEWS.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ Command-line option changes
 
 * In Linux and Windows, `--threads=auto` now tries to infer usable number of CPUs from the
   process affinity which is set typically in HPC and cloud environments ([#42340]).
+* `--math-mode=fast` is now a no-op ([#41638]). Users are encouraged to use the @fastmath macro instead, which has more well-defined semantics."
```
Comment (ericphanson) [inline NEWS.md:27]:
```suggestion
* `--math-mode=fast` is now a no-op ([#41638]). Users are encouraged to use the @fastmath macro instead, which has more well-defined semantics.
```

**Comment by PallHaraldsson** [comment]
Hi @ViralBShah since you added backport I assumed this would go in. "The underlying PR" my (trivial, not adding feature just dropping functionality for safety) PR #41638 was merged since Apr 14, I'm just unclear how to add it "on 1.8".

**Comment by ViralBShah** [comment]
Seems like I added the label mistakenly - if the PR is not on 1.8, we can't have the news there, naturally. It will just make it into 1.9.

**Pull request merged**